modules/site-health/audit-full-page-cache/helper.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * Helper functions used by module.
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Returns a list of headers and its verification callback to verify if page cache is enabled or not.
+ *
+ * Note: key is header name and value could be callable function to verify header value.
+ * Empty value mean existence of header detect page cache is enabled.
+ *
+ * @return array List of client caching headers and their (optional) verification callbacks.
+ * @since 1.0.0
```

Comment:
`@since` should come in its own block, _before_ `@param` and `@return`. Also, we need to use `n.e.x.t`.
```suggestion
 * @since n.e.x.t
 *
 * @return array List of client caching headers and their (optional) verification callbacks.
```